app/uk/gov/hmrc/gform/testonly/SnapshotADTs.scala

```diff
@@ -26,7 +26,10 @@ case class Snapshot(
   templateId: String,
   snapshotId: String,
   savedAt: Instant,
-  description: String
+  description: String,
+  gformVersion: String,
+  gformFrontendVersion: String,
+  hasTemplate: Boolean
```

Comment:
This feels like we are creating tech debt in snapshot logic. Snapshot should always have a type.

I think we should migrate user data instead. (Or remove existing snapshots after user consent)